plugins/uploadimage/plugin.js

```diff
@@ -146,5 +152,11 @@
 	 * @since 4.5
 	 * @cfg {String} [imageUploadUrl='' (empty string = disabled)]
 	 * @member CKEDITOR.config
+	 *
```

Comment:
Every config variable should have dedicated jsDoc block. Merging them is not allowed and not supported by jsduck.